moveit_core/CMakeLists.txt

```diff
@@ -128,9 +127,10 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
   Eigen3
   eigen3_cmake_module
   OCTOMAP
-  ${BULLET_ENABLE}
```

Comment:
Bullet needs to be listed as include depend, no?

Reply:
I think I omitted it because I was mirroring the FCL setup.  In fact the FCL include dependency is not in this list but I can add the Bullet one if that's what's preferred.